packages/core/src/models/extensions/dataSource.ts

```diff
@@ -7,6 +7,14 @@ import { InternalError } from '../../lib/utils/errors';
 import { ExtensionBase } from './base';
 import { VulcanExtension } from './decorators';
 
+export interface ExportOptions {
+  // The sql query result to export
+  sql: string;
+  // The full pathname to export result to file
+  directory: string;
+  // The profile name to select to export data
+  profileName: string;
```

Comment:
Could you assist to update and async the `exportOptions` to prevent solve the conflict when rebase the code, thanks:

```ts
export interface ExportOptions {
  // The sql query result to export
  sql: string;
  // The directory to export result to file
  directory: string;
  // The profile name to select to export data
  profileName: string;
  // export file format type
  type: CacheLayerStoreFormatType | string;
}
```